weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/likelyTimestampName.test.ts

```diff
@@ -0,0 +1,59 @@
+import {likelyTimestampName} from './ValueViewNumberTimestamp';
```

Comment:
I might suggest this filename either match ValueViewNumberTimestamp or we pull the related functions out into a matching .ts file.